src/command_modules/azure-cli-acs/HISTORY.rst

```diff
@@ -6,6 +6,9 @@ Release History
 2.0.33
 ++++++
 * fix typo in help message
+* make --connector-name optional for `aks install-connector`, `aks upgrade-connector` and `aks remove-connector`
```

Comment:
I think you'll have to put these bullet points in a `2.0.34` section since `2.0.33` has shipped now.